spec.md

```diff
@@ -814,6 +814,25 @@ So a pitch of `5` represent an `F4` and a pitch of `-2` represents an `A#3`.
 >
 > The pitches in this paragraph use [scientific pitch notation](https://en.wikipedia.org/wiki/Scientific_pitch_notation).
 
+Tools and games are expected to allow spaces to separate words both before and after the note text.
```

Comment:
Maybe we could add an indication that this part is now about the note text and not the pitch anymore? Something like:

```suggestion
The text of a note contain the spoken words of that note (usually a single syllable). There are no restrictions on the specific text values, in particular spaces can appear at the beginning or end of the note text (or both).
```

If that doesn't sound good, I'd at least prefer that we stay with the terminology "implementation" instead of "tools and games".